### ISSUE TITLE
Space removed before C# attributes on property getters

### ISSUE THREAD

**Issue by mihaipopescu** [uncrustify/uncrustify#1276]
```extern internal bool canAccess { [NativeMethod(Name = "CanAccessFromScript")] get; }```
gets formatted to
```extern internal bool canAccess {[NativeMethod(Name = "CanAccessFromScript")] get; }```
[space missing]

 ```extern public int subMeshCount { get; [NativeMethod<...>```
is kept intact though

Expected versus output diff:
---
```diff
diff --git "a/C:\\work\\external\\uncrustify\\tests\\output\\staging\\60024-UNI-19644.cs" "b/C:\\work\\external\\uncrustify\\tests\\results\\staging\\60024-UNI-19644.cs"
index da76bfb..c717992 100644
--- "a/C:\\work\\external\\uncrustify\\tests\\output\\staging\\60024-UNI-19644.cs"
+++ "b/C:\\work\\external\\uncrustify\\tests\\results\\staging\\60024-UNI-19644.cs"
@@ -1,7 +1,7 @@
 //It deletes the space after {
-class Foo 
+class Foo
 {
-    extern internal bool canAccess { [NativeMethod(Name = "CanAccessFromScript")] get; }
+    extern internal bool canAccess {[NativeMethod(Name = "CanAccessFromScript")] get; }
 
-    extern public int subMeshCount { get; [NativeMethod(Name = "CanAccessFromScript")] set; } 
+    extern public int subMeshCount { get; [NativeMethod(Name = "CanAccessFromScript")] set; }
 }
```

Debug support files:
---
[60024-UNI-19644.cs](https://gist.githubusercontent.com/mihaipopescu/f4cdde9d16e4e7ef4c11618f0e177d10/raw/da76bfbc696d8be1758b8a92f4bc3ff2e140cf54/60024-UNI-19644.cs)
[60024-UNI-19644.cs.log](https://gist.githubusercontent.com/mihaipopescu/f4cdde9d16e4e7ef4c11618f0e177d10/raw/7cf623757beec9fdf4c3e8981c3cf0e50d1306df/60024-UNI-19644.cs.log)
[60024-UNI-19644.cs.unc](https://gist.githubusercontent.com/mihaipopescu/f4cdde9d16e4e7ef4c11618f0e177d10/raw/c82cb342c69648e9286c215569fc980613e0f9ff/60024-UNI-19644.cs.unc)
[Uncrustify.CSharp.cfg](https://gist.githubusercontent.com/mihaipopescu/f4cdde9d16e4e7ef4c11618f0e177d10/raw/1d064c1fabb76ce328dd015ed40023207de20ca1/Uncrustify.CSharp.cfg)
[UNI-19644.cs](https://gist.githubusercontent.com/mihaipopescu/f4cdde9d16e4e7ef4c11618f0e177d10/raw/da76bfbc696d8be1758b8a92f4bc3ff2e140cf54/UNI-19644.cs)

Disclaimer:
---
Note: This is a generated issue from our internal issue tracker.
Expect a dry description which could be insufficient to properly understand the issue.
On the other hand, the support files should be sufficient to easily reproduce and debug the issue,
but please comment if it's still unclear and I will get back with clarifications.
See main discussion thread here #1223.

**Comment by CDanU** [comment]
Caused by:
```sql
# Add or remove space before '[' (except '[]')
sp_before_square                = remove   # ignore/add/remove/force
```

**Comment by satishp87** [comment]
In unity fork this issue is addressed with below PR
https://github.com/Unity-Technologies/uncrustify/pull/27

**Comment by gmaurel** [comment]
Fixed at db6adb6c69b238746c60233a423eac3df02c4073, Dec 20 20:25:54 2017
